types/src/front/assistant/conversation.ts

```diff
@@ -219,3 +219,15 @@ export interface ConversationParticipantsType {
   agents: AgentParticipant[];
   users: UserParticipant[];
 }
+
+export class ConversationPermissionError extends Error {
```

Comment:
Let's not introduce one error per case, we have a generic `DustError` that can be used for those two use cases.